darts/ad/anomaly_model/anomaly_model.py

```diff
@@ -247,6 +247,7 @@ def show_anomalies(
         names_of_scorers: Union[str, Sequence[str]] = None,
         title: str = None,
         metric: Optional[Literal["AUC_ROC", "AUC_PR"]] = None,
+        multivariate_plot: bool = False,
```

Comment:
Please update the docstring to include a description of what this argument is doing